manifests/gemrc.pp

```diff
@@ -33,32 +33,19 @@
 #    }
 
 class ruby::gemrc (
-  $sources                    = undef,
-  $verbose                    = undef,
-  $update_sources             = undef,
-  $backtrace                  = undef,
-  $gempath                    = undef,
-  $gem_command                = undef,
-  $gemrc                      = $::ruby::params::gemrc,
-  $owner                      = 'root',
-  $group                      = 'root',
-  $mode                       = '0644',
-  $disable_default_gem_server = undef
+  $sources                                              = undef,
+  Optional[Variant[Boolean, Enum[':really']]] $verbose  = undef,
+  Optional[Boolean] $update_sources                     = undef,
+  Optional[Boolean] $backtrace                          = undef,
+  $gempath                                              = undef,
+  $gem_command                                          = undef,
+  $gemrc                                                = $::ruby::params::gemrc,
+  $owner                                                = 'root',
+  $group                                                = 'root',
```

Comment:
These should all just be `String`, except for `gem_command`, which should be `Optional[String]`